static/bidder-info/mediafuse.yaml

```diff
@@ -1,17 +1,15 @@
 maintainer:
   email: "support@mediafuse.com"
-gvlVendorID: 411
 capabilities:
   app:
     mediaTypes:
       - banner
       - video
+      - native
   site:
     mediaTypes:
       - banner
       - video
+      - native
 userSync:
-  # mediafuse supports user syncing, but requires configuration by the host. contact this
-  # bidder directly at the email address in this file to ask about enabling user sync.
-  supports:
-    - iframe
\ No newline at end of file
+  key: "adnxs"
```

Comment:
This aliases the AppNexus cookie sync for use by this adapter as well. Ensures you'll still get a cookie match. Please confirm this is desired, or if you have your own cookie sync process.

Reply:
we are okay with appnexus cookie sync